packages/functions/src/utilities/fullTestUtilities.ts

```diff
@@ -148,6 +147,21 @@ export interface SpeakingAnswer {
       questions: string[];
     };
   };
-  feedback?: any; // SpeakingFeedback
+  feedback?: SpeakingFeedbackAll; // SpeakingFeedback
   status: FeedbackStatus;
 }
+
+export type SpeakingFeedbackAll = (SpeakingFeedback | Error)[];
+
+export type WritingFeedbackAll = (WritingFeedback | Error)[];
+
+export type RLFeedbackAll =
+  | {
+      CorrectAnswers: any[]; //allCorrectAnswers;
+      studentAnswers: string[] | string[][]; //studentAnswers;
```

Comment:
You have incorrect type.  It is `(string[] | string[][])[][]`.  You might as well just use `any`.

One could argue that it makes more sense to store answers as an array of 40 strings, but we didn't do that.  You have to live with that :).

Another question.  Does the front end anticipate a parameter of a similar structure?